website/content/docs/connect/cluster-peering/usage/create-sameness-groups.mdx

```diff
@@ -47,45 +43,38 @@ Mesh gateways are required for cluster peering connections and recommended to se
 
 You must establish connections with cluster peers before you can create a sameness group that includes them. A cluster peering connection exists between two admin partitions in different datacenters, and each connection between two partitions must be established separately with each peer. Refer to [establish cluster peering connections](/consul/docs/connect/cluster-peering/usage/establish-cluster-peering) for step-by-step instructions.
 
-To establish cluster peering connections and define a define group as part of the same workflow, follow instructions up to [Export services between clusters](/consul/docs/connect/cluster-peering/usage/establish-cluster-peering#export-services-between-clusters). You can use the same exported services and service intention configuration entries to establish the cluster peering connection and create the sameness group.
+You can establish and manage cluster peering relationships between all of your self-managed clusters using [HCP Consul Central](/hcp/docs/consul/concepts/consul-central). For more information, refer to [cluster peering global view](/hcp/docs/consul/monitor/consul-central/global-views#cluster-peering) in the HCP documentation.
+
+To establish cluster peering connections and define a group as part of the same workflow, follow instructions up to [Export services between clusters](/consul/docs/connect/cluster-peering/usage/establish-cluster-peering#export-services-between-clusters). You can use the same exported services and service intention configuration entries to establish the cluster peering connection and create the sameness group.
 
 ## Create a sameness group
 
-To create a sameness group, you must write and apply sets of three configuration entries for each partition that is a member of the group:
+The workflow to create a sameness group requires you to write and apply a set of three configuration entries for each partition that is a member of the group:
```

Comment:
each partition or peer

I think in this doc we kinda refer to peers as configuring a sameness group with remote partitions?

Reply:
@hashi-derek @erichaberkorn do you find that the distinction between partition and peer is clear in this documentation in regards to sameness?

Reply:
I feel like it might be a little bit confusing for users unless they look at sameness group config entries up-front and study the fields on them. I do believe that the concept of mixing local / remote partitions together into the sameness group is probably still going to be unclear after following these docs, but perhaps that's something we should specify in the config-entry docs as prerequisite knowledge?

This tutorial seems to be specifically aimed at using SGs with cluster peering, and so it might be better to avoid overloading / confusing the reader with extra info. I'm not sure exactly what would be the smoothest path to onboarding users to this feature, but we probably do need to look at the entire journey they will take and try to steer them in the proper order of documents to study as best as we can.